test/e2e/data/infrastructure-vsphere-govmomi/v1.8/base/cluster-template-topology.yaml

```diff
@@ -103,20 +103,6 @@ spec:
     matchLabels:
       cluster.x-k8s.io/cluster-name: '${CLUSTER_NAME}'
   resources:
-  - kind: Secret
-    name: vsphere-csi-controller
-  - kind: ConfigMap
-    name: vsphere-csi-controller-role
-  - kind: ConfigMap
-    name: vsphere-csi-controller-binding
-  - kind: Secret
-    name: csi-vsphere-config
-  - kind: ConfigMap
-    name: csi.vsphere.vmware.com
-  - kind: ConfigMap
-    name: vsphere-csi-node
-  - kind: ConfigMap
-    name: vsphere-csi-controller
   - kind: Secret
     name: cloud-controller-manager
   - kind: Secret
```

Comment:
This seems to drop CSI instead of replacing the image. Is this because there are no new images available?

Reply:
This drops csi for the manifests of tests involving v1.8. these manifests still use v2.1 csi which  we don't have images for... also we don't depend on csi during these tests.